appveyor.yml

```diff
@@ -39,6 +39,12 @@ for:
     artifacts:
       - path: 'artifacts\*.nupkg'
       - path: 'artifacts\*.snupkg'
+    deploy:
+    - provider: NuGet
+      api_key:
+        secure: <encrypted-nuget-api-key>
```

Comment:
@axunonb could you please add the **encrypted** api key here?

Steps.
1. Create an API key: https://www.nuget.org/account/apikeys
2. Encrypt the API key here: https://ci.appveyor.com/account/YAXLib/tools/encrypt
3. Edit this file in this PR and replace `<encrypted-nuget-api-key>` with the encrypted API key

Reply:
@304NotModified done

Reply:
Thanks! Looks good!